src/Main/Tasks.cpp

```diff
@@ -88,7 +107,20 @@ auto synthStrToOpt = [](const QString& str) -> FOEDAG::Compiler::SynthesisOpt {
 
   return val;
 };
+auto placeStrToOpt = [](const QString& str) -> FOEDAG::Compiler::PlacementOpt {
+  auto it = find_if(
+      placeOptMap.begin(), placeOptMap.end(),
+      [str](const std::pair<FOEDAG::Compiler::PlacementOpt, const char*> p) {
+        return p.second == str;
```

Comment:
this will not work. you just compare the pointers but should be string comparison.